.github/workflows/cd.yml

```
@@ -3,20 +3,25 @@ name: Build and deploy to production
 on:
   push:
     branches: [main]
+  pull_request:
+    types: [closed]
+    branches: [main]
```

Dibikin folder tertentu aja mba biar semisa ada update di ningali yang jalan publishernya aja

```suggestion
    branches: [main]
    paths:
    - 'packages/server/**'
    - 'packages/shared/**'
  pull_request:
    types: [closed]
    branches: [main]
    paths:
    - 'packages/server/**'
    - 'packages/shared/**'
```